sdk/monitor/azure-monitor-opentelemetry/README.md

```diff
@@ -9,21 +9,24 @@ This distro automatically installs the following libraries:
 
 ## Officially supported instrumentations
 
-OpenTelemetry instrumentations allow automatic collection of requests sent from underlying instrumented libraries. The following is a list of OpenTelemetry instrumentations that come bundled in with the Azure monitor distro. If you would like to add support for another OpenTelemetry instrumentation, please submit a feature [request][distro_feature_request]. In the meantime, you can use the OpenTelemetry instrumentation manually via it's own APIs (i.e. `instrument()`) in your code. See [this][samples_manual] for an example.
-
-| Instrumentation | Supported library | Supported versions |
-| ------------------------------------- | ----------------- | ------------------ |
-| [OpenTelemetry Django Instrumentation][ot_instrumentation_django] | [django][pypi_django] | [link][ot_instrumentation_django_version]
-| [OpenTelemetry FastApi Instrumentation][ot_instrumentation_fastapi] | [fastapi][pypi_fastapi] | [link][ot_instrumentation_fastapi_version]
-| [OpenTelemetry Flask Instrumentation][ot_instrumentation_flask] | [flask][pypi_flask] | [link][ot_instrumentation_flask_version]
-| [OpenTelemetry Psycopg2 Instrumentation][ot_instrumentation_psycopg2] | [psycopg2][pypi_psycopg2] | [link][ot_instrumentation_psycopg2_version]
-| [OpenTelemetry Requests Instrumentation][ot_instrumentation_requests] | [requests][pypi_requests] | [link][ot_instrumentation_requests_version]
-| [OpenTelemetry UrlLib Instrumentation][ot_instrumentation_urllib] | [urllib][pypi_urllib] | All
-| [OpenTelemetry UrlLib3 Instrumentation][ot_instrumentation_urllib3] | [urllib3][pypi_urllib3] | [link][ot_instrumentation_urllib3_version]
+OpenTelemetry instrumentations allow automatic collection of requests sent from underlying instrumented libraries. The following is a list of OpenTelemetry instrumentations that come bundled in with the Azure monitor distro. These instrumentations are enabled by default. See the [Usage](#usage) section below for how to opt-out of these instrumentations.
+
+| Instrumentation | Supported library | Entry Point Name | Supported versions |
+| --------------- | ----------------- | ---------------- | ------------------ |
+| [Azure Core Tracing OpenTelemetry][azure_core_tracing_opentelemetry_plugin] | `Azure Core libraries` | `azure_sdk` | |
```

Comment:
Are we able to confirm if we can put "All" for supported azure sdk versions?

Reply:
This would be complicated since it's not just one package. I haven't found a clear answer in the azure core tracing docs. But, I could look more into it.